src/views/ModelPlan/Discussions/index.tsx

```diff
@@ -348,115 +349,6 @@ const Discussions = ({ modelID, askAQuestion, readOnly }: DiscussionsProps) => {
     );
   };
 
-  const discussionComponent = (
-    discussion: DiscussionType | ReplyType,
-    index: number,
-    connected?: boolean,
-    answerQuestion?: boolean
-  ) => (
-    <div className="mint-discussions__single-discussion" key={discussion.id}>
-      <div className="display-flex">
-        {discussion.isAssessment ? (
-          <div className="display-flex flex-align-center">
-            <AssessmentIcon size={3} />{' '}
-            <span>
-              {t('assessment')} | {discussion.createdBy}
-            </span>
-          </div>
-        ) : (
-          <IconInitial
-            user={
-              collaborators.find(
-                collaborator => collaborator.euaUserID === discussion.createdBy
-              )?.fullName || ''
-            }
-            index={index}
-            className="margin-bottom-2"
-          />
-        )}
-        <span className="margin-left-2 margin-top-05 text-base">
-          {getTimeElapsed(discussion.createdDts)
-            ? getTimeElapsed(discussion.createdDts) + t('ago')
-            : t('justNow')}
-        </span>
-      </div>
-
-      <div
-        className={classNames({
-          'margin-bottom-4': answerQuestion,
-          'mint-discussions__connected': connected,
-          'mint-discussions__not-connected': !connected
-        })}
-      >
-        <p className="margin-y-0 padding-y-1">{discussion.content}</p>
-        <div className="display-flex margin-bottom-2">
-          {/* Rendered a link to answer a question if there are no replies/answers only for Collaborator and Assessment Users */}
-          {hasEditAccess && answerQuestion && (
-            <>
-              <IconAnnouncement className="text-primary margin-right-1" />
-              <Button
-                type="button"
-                unstyled
-                role="button"
-                onClick={() => {
-                  setDiscussionStatusMessage('');
-                  setDiscussionType('reply');
-                  setReply(discussion);
-                }}
-              >
-                {t('answer')}
-              </Button>
-            </>
-          )}
-        </div>
-      </div>
-    </div>
-  );
```

Comment:
This function is replaced with `SingleDiscussion` component.
src/views/ModelPlan/Discussions/SingleDiscussion.tsx